projects/plugins/boost/app/modules/optimizations/page-cache/data-sync-actions/deactivate-wpsc.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Automattic\Jetpack_Boost\Modules\Optimizations\Page_Cache\Data_Sync_Actions;
+
+use Automattic\Jetpack\WP_JS_Data_Sync\Contracts\Data_Sync_Action;
+
+/**
+ * Cache Action: Disable Super Cache
+ */
+class Deactivate_WPSC implements Data_Sync_Action {
+
+	/**
+	 * Handles the action logic.
+	 *
+	 * @param mixed                 $_data    JSON Data passed to the action.
+	 * @param null|\WP_REST_Request $_request The request object.
+	 */
+	public function handle( $_data = null, $_request = null ) {
+		// Super Cache will define WPCACHEHOME if it's active.
+		if ( ! defined( 'WPCACHEHOME' ) ) {
+			return true;
+		}
```

Comment:
Are we sure super cache is the only plugin that defines this constant?

Reply:
I would say that yes, it is the only plugin using that constant. It's defined every time WPSC is activated and I've never had a complaint that it was already defined.